rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -32,7 +32,7 @@ function getGridLines(lowerGridBound = -10, upperGridBound = 10) {
     }
 
     var finalLineGridSet = []
-    coordPairSet.map(coordPair => {
+    coordPairSet.forEach(coordPair => {
```

Comment:
out of curiosity, whats the difference between `map` and `forEach`?

Reply:
`map` requires a return type, but in our case, we don't want anything returned in this for loop. So `forEach` is the more appropriate function to use, and the one I should have been using from the start.